.circleci/config.yml

```diff
@@ -535,9 +535,11 @@ commands:
       - run:
           name: Upload build artifacts to central repository (AWS)
           command: |
-            envdir /secrets aws s3 cp --acl public-read ${PACKAGE_TARBALL:?} s3://${S3_BUILDS_BUCKET:?}/
-            if [ -n "$CIRCLE_BRANCH" -a "$CIRCLE_BRANCH" = "master" ]; then
-              envdir /secrets aws s3 cp --acl public-read ${PACKAGE_TARBALL:?} s3://${S3_BUILDS_BUCKET:?}/aeternity-latest-${PKG_SUFFIX:?}${PKG_EXT:-".tar.gz"}
+            if [ -z "$CIRCLE_PR_NUMBER" ]; then
```

Comment:
How does this change how the CI works for forks?

Reply:
This effectively disables the jobs that require secrets during the builds, as the secrets are not shared with the forks, thus otherwise it will fail.

The actual option to build the forks is already enabled in CircleCI UI.

Reply:
Ok that piece of information was missing :-) Thanks.